jvm-packages/project/Common.scala

```diff
@@ -0,0 +1,22 @@
+import sbt._
+import Keys._
+
+object Common {
+  val appVersion = "0.7"
+
+  val settings: Seq[Def.Setting[_]] = Seq(
+    version := appVersion,
+    organization := "ml.dmlc",
+
+    crossScalaVersions := Seq("2.10.6", "2.11.11"),
+
+    javacOptions ++= Seq("-source", "1.7", "-target", "1.7"),
+    scalacOptions ++= Seq("-deprecation", "-unchecked"),
+
+    resolvers += Opts.resolver.mavenLocalFile,
+    resolvers ++= Seq(DefaultMavenRepository,
+      Resolver.defaultLocal,
+      Resolver.mavenLocal,
+      Resolver.jcenterRepo)
+  )
+}
```

Comment:
can you check if scala-style plugin covers these files? I have an impression that file must be ended with new line in the default setup

Reply:
I think it doesn't because otherwise, it would enforce a header. But that's a good point, will do.

Reply:
Do you know how to do this? Scalstyle docs are mute on the subject, sadly.

Reply:
I think we can set something in https://github.com/dmlc/xgboost/blob/master/jvm-packages/pom.xml#L59?